rollup/tracing/tracing.go

```diff
@@ -323,6 +342,8 @@ func (env *TraceEnv) getTxResult(state *state.StateDB, index int, block *types.B
 	if err != nil {
 		return err
 	}
+	getTxResultApplyMessageTimer.UpdateSince(applyMessageStart)
```

Comment:
Should we also add this inside the `err != nil` branch?

Reply:
add [586a283](https://github.com/scroll-tech/go-ethereum/pull/700/commits/586a28308a30bca6ead75722d6136533abf012a0)